lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/json/JsonDeserializer.java

```diff
@@ -352,10 +354,8 @@ void decodeValue(JsonParser parser, BlockBuilder builder)
             try {
                 bigDecimal = HiveFormatUtils.parseDecimal(value, decimalType);
             }
-            catch (NumberFormatException _) {
-                // parsing errors are simply ignored and the field is null
-                builder.appendNull();
-                return;
+            catch (NumberFormatException e) {
+                throw new TrinoException(BAD_DATA, "Error Parsing a column in the table: " + e.getMessage(), e);
```

Comment:
Should we throw `TrinoException(BAD_DATA, ...)` on overflow too on [line 362](https://github.com/trinodb/trino/pull/22513/files#diff-e02c43d19cafaa78cf36f09bc154c7a14730cdb27282c859db2b97251c1917b7R362)? It seems like these two scenarios should be handled in a consistent manner.

Reply:
Yeah, good point. I will make it so that the TrinoException(BAD_DATA, ...) wraps the NumberFormatException. I think the context of the NumberFormatException is still good to include, but agree with wrapping in the trino exception.